mica/archive/aca_l0.py

```diff
@@ -35,11 +36,11 @@
                       'tlmver', 'ascdsver', 'revision', 'date',
                       'imgsize')
 
-
-FDTYPE = [('level', '|S4'), ('instrum', '|S7'), ('content', '|S13'),
-         ('arc5gl_query', '|S27'), ('fileglob', '|S9')]
-FILETYPE = np.rec.fromrecords([('L0', 'PCAD', 'ACADATA', 'ACA0', '*fits.gz')],
-                              dtype=FDTYPE)[0]
+FILETYPE = {'level': 'L0',
```

Comment:
The np.rec.fromrecords gave us unnecessary bytestrings.  I *think* this needs no new comments as the "this used to be bytestrings" part isn't interesting and the simple dictionary to store the type seems self-documenting.